packages/web-components/src/components/tabs-extended/tabs-extended.ts

```diff
@@ -88,7 +88,33 @@ class DDSTabsExtended extends MediaQueryMixin(
 
   private _handleAccordionClick(e) {
     const tab = e.target.closest(`${ddsPrefix}-tab`);
+    const currentIndex = this._activeTabIndex;
+
+    // Adding the ability to close the accordion. If the tab that's clicked is already active, we pass a bogus number through _setActiveItem() method, so that all tabs are closed.
+    if (tab.getIndex() === currentIndex && !this._isLargeOrGreater) {
+      this._setActiveItem(-1);
+    }
+
+    // If the tab is already active and selected when clicked, just return
+    if (
+      tab &&
+      tab.getIndex() === currentIndex &&
+      this._tabItems[currentIndex].selected
+    ) {
+      return;
+    }
+
     this._handleClick(tab.getIndex(), e);
+
+    // Checking if the clicked tab is different from the previously active tab, if they're different, it just sets the focus to the new clicked tab
+    if (currentIndex !== tab.getIndex()) {
+      const tabButton = tab.shadowRoot?.querySelector(
+        `.${prefix}--accordion__heading`
+      );
+      if (tabButton instanceof HTMLElement) {
+        tabButton.focus();
+      }
+    }
```

Comment:
I'm also not sure you need this! We're still setting focus in the `_setActiveItem` method.

Reply:
Code's been remove as suggested and it's working! Can you please review it now @jkaeser